### PR TITLE
#admin/1669 Exercise Dashboard

### PR DESCRIPTION
**Pull request by lloback**
## What's included?
This PR has the development work for the Dashboard. This is a new functionality for the exercise page.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
URL: https://jac-admin-develop--pr1679-1669-dashboard-gcwx9xkb.web.app/

Just check the New DASHBOARD page.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
This is still on development

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_

### CODE REVIEW

**Review by warrensearle — APPROVED** [review]
Looks good to me 👍

**Review by andrew-isaac — APPROVED** [review]
I have sent to Rob, Matt and Sean for comments - looks good to me, fantastic work :)

File: src/views/Exercises.vue [hunk 1]
```diff
@@ -85,28 +85,28 @@
             <template #row="{row}">
               <TableCell :title="tableColumns[0].title">
                 <RouterLink
-                  :to="{ name: 'exercise-overview', params: { id: row.id } }"
+                  :to="{ name: 'exercise-dashboard', params: { id: row.id } }"
```
Comment (tomlovesgithub) [inline src/views/Exercises.vue:88]: i think these should still link to the overview page, as on just created exercises the overview appears blank.

**Comment by PShanYu** [comment]
UI Improvement Suggestion (1):

Make the box same height, so the dashboard can be visually more organised : ) 

Screenshot 2022-09-20 at 15 40 19

**Comment by PShanYu** [comment]
UI Improvement Suggestion (2):

Make the font size of the most important info larger to create a better visual hierarchy : )

Screenshot 2022-09-20 at 15 46 58

Screenshot 2022-09-20 at 15 43 04

**Comment by PShanYu** [comment]
UI Improvement Suggestion (3):

I recommend that we can use the chevron icon from UK.GOV design system if it is available. This version is smaller and simpler, so it creates less visual distraction for users.  

Screenshot 2022-09-20 at 15 54 17

Screenshot 2022-09-20 at 15 51 34

Screenshot 2022-09-20 at 15 52 56

**Comment by PShanYu** [comment]
UI Improvement Suggestion (4):

Put the categories of chart in order, so users can find the information much easier : )

I also recommend that we can add number for each categories. It will also help users, especially who are struggling with recognising colours, find the info easier : )

Screenshot 2022-09-20 at 16 26 16

Screenshot 2022-09-20 at 17 21 04

**Comment by PShanYu** [comment]
UI Improvement Suggestion (5):

We can make the space between two bars wider. It also help users to read the info : )

Screenshot 2022-09-20 at 17 25 31

Screenshot 2022-09-20 at 17 26 43

**Comment by PShanYu** [comment]
UI Improvement Suggestion (7):

For the initial status of dashboard, my suggestion is as the attached image : ) 

Screenshot 2022-09-20 at 16 55 19

V1 Initial

**Comment by PShanYu** [comment]
UI Improvement Suggestion (8):

This section is missing from the current dashboard.

Screenshot 2022-09-20 at 17 41 42

**Review by HalcyonJAC — APPROVED** [review]
LGTM 👍

**Comment by warrensearle** [comment]
Closing as this work is now part of #1786

**Pull request closed without merging**